platformio.ini

```diff
@@ -33,7 +33,7 @@ board_build.f_cpu = 240000000L
 ; BZZZ_DEBUG should be used only for debugging
 build_flags =
 	-D BZZZ_DEBUG
-	-D BZZZ_LOGGING_LEVEL=3
+	-D BZZZ_LOGGING_LEVEL=2
```

Comment:
I would change this back to 

```c++
build_flags =
	-D BZZZ_DEBUG
	-D BZZZ_VERBOSITY=3
```

before merging

Reply:
Done.